### ISSUE TITLE
Very high memory and CPU usage

### ISSUE THREAD

**Issue by culminis1** [mwarning/zerotier-openwrt#105]
I was using Zerotier on 4 [GL.inet GL-MIFI](https://www.gl-inet.com/products/gl-mifi/) openwrt routers. I have observed high amounts of memory and CPU usage, over 50%. Sometimes the routers crashes.
Do you noticed that? What could be the origin?

- Zerotier version 1.6.4
- OpenWrt version  19.07.8
- Zerotier router interface bridged (network segment 192.168.1.0/24) with br-lan (with o without IGMP Snooping and STP, both the same results)
- Zerotier central network: bridges allowed, no-assigning IPs, default route [ 192.168.1.0/24 => (LAN) ]

Any help will be welcome.
Thanks

**Comment by mwarning** [comment]
Please try OpenWrt master (uses ZeroTier 1.10.1) and see if the problem is still present?
You can download an image here https://firmware-selector.openwrt.org
Either use the package customization to add ZeroTier or use `opkg update` and `opkg install zerotier`

**Comment by culminis1** [comment]
First of all, thank you for your quick answer.
Despite I'm developer in other environments, I don't have many skills in unix/linux/openWrt systems. I haven't the toolchain for building my own openWrt image. I am using the GLinet manufacturer stable image, and installing the zerotier package on it. Is it a right procedure for testing your proposition?
I will try to install the 1.10.1 version on the current GLinet openWrt image, but I can not find the 1.10.1 version in OpenWrt package repositories, none higher than 1.8.6-2.
I have tried to install Zerotier version 1.8-6-2. Installation complaints against the libnatpmp1 absence, but libnatpmp is installed.
I've updated manually from the same repo than 1.8 zerotier,   and when the service starts through errors libe _"libminiupnpc.so.17: __clock_gettime64: symbol not found"_

How could address the problem?

Thank you again for your suporte
JL
